src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs

```diff
@@ -78,6 +82,12 @@ public static class ControllerConstants
         public const string ViewAccountsActionName = "ViewAccounts";
         public const string ViewTeamActionName = "ViewTeam";
         public const string ViewTeamViewName = "ViewTeam";
+        public const string WhichCourseYourApprenticeWillTakeActionName = "WhichCourseYourApprenticeWillTake";

```

Comment:
Can all these partials be renamed with a a prefix so that they are grouped together?  e.g.  TriageYouCannotSetupAnApprenticeshipYetCourseProviderActionName, TriageYouCannotSetupAnApprenticeshipYetProviderActionName 
So it's clear that they are related.

Reply:
Done